requirements.txt

```diff
@@ -0,0 +1,13 @@
+autopep8==1.5.4
+click==7.1.2
+Flask==1.1.2
+Flask-Cors==3.0.9
+Flask-Limiter==1.4
+itsdangerous==1.1.0
+Jinja2==2.11.2
+limits==1.5.1
+MarkupSafe==1.1.1
+pycodestyle==2.6.0
+six==1.15.0
+toml==0.10.2
+Werkzeug==1.0.1
```

Comment:
There's a lot of packages in this requirements.txt that don't appear relevant.

Reply:
I have removed the dependency libraries.